src/main/java/com/google/api/codegen/viewmodel/StaticLangApiSchemaView.java

```diff
@@ -0,0 +1,90 @@
+package com.google.api.codegen.viewmodel;
+
+import com.google.api.codegen.SnippetSetRunner;
+import com.google.api.codegen.discovery.Schema;
+import com.google.auto.value.AutoValue;
+import java.util.List;
+import javax.annotation.Nullable;
+
+/**
+ * This ViewModel defines the structure of a Discovery doc's "schemas", "properties",
+ * "additionalProperties", and "items".
+ *
+ * This contains a subset of properties in the JSON Schema
+ * https://tools.ietf.org/html/draft-zyp-json-schema-03#section-5.7.
+ */
+@AutoValue
+public abstract class StaticLangApiSchemaView implements ViewModel {
```

Comment:
I would suggest `StaticLangMessageView` (name the file view one in a corresponding way)